lib/mapObjects/CGMarket.h

```diff
@@ -75,7 +74,12 @@ class DLL_LINKAGE CGBlackMarket : public CGMarket
 class DLL_LINKAGE CGUniversity : public CGMarket
 {
 public:
+	int skillsTotal = 4;
```

Comment:
Perhaps simply use value provided in config?
If config defines 4 skills - then university will have 4 skills. If number is different - then well, university will also have different number of skills.

Reply:
actually, this one is even not needed, I believe since skills is an array